manim/mobject/text/numbers.py

```diff
@@ -214,17 +226,21 @@ def _get_num_string(self, number):
 
         return num_string
 
-    def _string_to_mob(self, string: str, mob_class: VMobject | None = None, **kwargs):
+    def _string_to_mob(
+        self, string: str, mob_class: type[TextLike] | None = None, **kwargs: Any
+    ) -> TextLike:
         if mob_class is None:
             mob_class = self.mob_class
 
+        _mob_class = self.mob_class if mob_class is None else mob_class
```

Comment:
This feels a little bit weird - does the extra statement do anything?

Reply:
I could swear that MyPy complained to me that `mob_class` could be `None` if I didn't do that, but now I removed it and it's OK with that...?

BTW, thanks for noticing - it didn't make sense that both that extra statement and the lines 232 and 233 existed at the same time, anyways.